atomic_reactor/plugins/post_generate_maven_metadata.py

```diff
@@ -56,6 +67,90 @@ def get_nvr_components(self, nvr_requests):
 
         return components
 
+    def download_sources(self, url_requests):
+        download_queue = []
+        source_url_to_artifacts = {}
+        remote_source_files = []
+        session = util.get_retrying_requests_session()
+
+        for url_request in url_requests:
+            if 'source-url' not in url_request:
+                msg = 'fetch-artifacts-url without source-url is deprecated'
+                self.log.warning(msg)
+                self.log.user_warning(msg)
+                return []
```

Comment:
shouldn't this be 'continue' ?

and we should also log, for which URL is missing source-url

Reply:
Fixed it. We'll fetch all sources we can find and we'll add a 'no_source' field that'll list all the artifacts for which we couldn't find sources.